hijack/tests/test_app/settings.py

```diff
@@ -100,4 +101,5 @@
 
 AUTH_USER_MODEL = "test_app.CustomUser"
 
-SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
+if django.VERSION < (4, 0):
+    SESSION_SERIALIZER = "django.contrib.sessions.serializers.PickleSerializer"
```

Comment:
Is this actually needed? All Django versions seem to default to the JSON serializer, so I'd drop it altogether.

Reply:
I tried but tests were not passing on Django 3.2 with the Json serializer...

Reply:
That's interesting. I wonder why this would happen during testing only, as I expect nobody changes the default `SESSION_SERIALIZER`.

Reply:
I investigated a bit and found that the failing test is altering a date in the session:
```
        expire_date = timezone.now() + datetime.timedelta(hours=30)
        session.set_expiry(expire_date)
```
What is weird is that only 3.2 is failing...

Reply:
I appreciate your looking into it. If there's a plan to eventually drop Django 3.2, then I'd say it's not worth investigating it any further.

Maybe the authors of the library could share their insights and opinions.

Reply:
What I considered to pick this solution:
- the proper setter is used in the test
- 3.2 end of life is in 3 months
- Pickle is the default serializer on this version
- it's working fine with the default serializer of each version
- (I never switched this setting on any project)